README.md

```diff
@@ -1,2 +1,44 @@
 # Hello-coffee-
+Task 1
+Machine Have Needs!
+Solution: Class MachineHaveNeeds
+
```

Comment:
@GayaneGulanyan jan will be good to see why you designed in this way

Reply:
In case you asked why Class is along and doesn't used, I didn't find useful to include it in the second task. Because it's output was not useful for the second task.
But as we has a project I mentioned it in the README as the part of our project.